easybuild/tools/testing.py

```diff
@@ -295,7 +295,7 @@ def post_pr_test_report(pr_nr, repo_type, test_report, msg, init_session_state,
 
     if build_option('include_easyblocks_from_pr'):
         if repo_type == GITHUB_EASYCONFIGS_REPO:
-            easyblocks_pr_nrs = [int(pr_nr) for pr_nr in build_option('include_easyblocks_from_pr')]
+            easyblocks_pr_nrs = [int(eb_pr_nr) for eb_pr_nr in build_option('include_easyblocks_from_pr')]
```

Comment:
A safer fix would be using `for x in `, since we don't use single-letter variables outside of list comprehensions... :)

Reply:
Also, can we come up with a way to make the tests trip over this?

Reply:
done in #3726